### ISSUE TITLE
Log queries

### ISSUE THREAD

**Issue by vespino** [klutchell/unbound-docker#351]
I’m looking to block a number of pop-up websites and I was wondering if it’s possible to log DNS queries so I can find the exact links.

**Comment by klutchell** [comment]
You can try increasing the [verbosity](https://unbound.docs.nlnetlabs.nl/en/latest/manpages/unbound.conf.html) value with a custom config file. That doesn't write the logs to file but it does write more to stdout.

Otherwise there might be another supported unbound setting to get what you need.

Also, running Pi-hole in front of unbound is a popular solution for blocking ads and popups. Here is an [example](https://github.com/klutchell/unbound-docker/blob/main/examples/pi-hole/docker-compose.yml).

**Comment by vespino** [comment]
I opted for Unbound instead of Pi-hole because of its simplicity. A simple file already blocks ads for me.

**Comment by klutchell** [comment]
Fair enough, no argument there. I think the best way at this time is to increase the verbosity of the unbound logs, as the default is currently set to 1.

There may be other methods by digging through the official docs, but I've tried nothing like that personally.
